src/frontend-build.sh

```diff
@@ -205,6 +206,9 @@ if running_in_ci && ! setup_docker_login; then
     exit 1
 fi
 
+# Caller might not want to run verification, just build the image. Set to the default value of $IS_PR
+export RUN_VERIFY=${RUN_VERIFY:-$IS_PR}
```

Comment:
following my previous comment, can we make this a flag to skip, instead of to run the verify task if used ?

this way I´d use something like

```
export SKIP_VERIFY=$SKIP_VERIFY:-}
```